epilepsy12/management/commands/seed.py

```diff
@@ -168,6 +168,9 @@ def run_dummy_cases_seed(cases, organisations, noskip, verbose=True):
         num_cases_to_seed_in_org = int(cases / len(organisations))
         print(f"Creating {num_cases_to_seed_in_org} Cases in {org}")
 
+        # Check if the organisation is in Jersey - this is important for generating postcodes and URNs rather than NHS Numbers
+        is_jersey = org.country.boundary_identifier == "JEY"
```

Comment:
We should add Jersey to the ORGANISATION_SEED_LIST so we have patients there in our local dev instances to tease out any unexpected bugs

Reply:
Shazam!